PyInstaller/hooks/hook-accessible_output2.py

```diff
@@ -0,0 +1,10 @@
+"""
+accessible_output2: http://hg.q-continuum.net/accessible_output2
+"""
+
+import os.path
+import accessible_output2 as _accessible_output2
+
+_dir = os.path.dirname(_accessible_output2.__file__)
+
+binaries = [(os.path.join(_dir, 'lib'), os.path.join('accessible_output2', 'lib'))]
```

Comment:
This would be easier to get if written like this:
```
binaries = [(os.path.join(os.path.dirname(_accessible_output2.__file__), 'lib'),
             os.path.join('accessible_output2', 'lib'))]
```

But it would be even better to use `collect_dynamic_libs( 'module-name' )`, see https://pyinstaller.readthedocs.io/en/stable/hooks.html#useful-items-in-pyinstaller-utils-hooks